ReactAndroid/gradle.properties

```diff
@@ -1,5 +1,5 @@
 # TODO :: ANDROID patch to stay consistant with the npm version.
-VERSION_NAME=0.63.3
+VERSION_NAME=0.0.0-8ed0e0e77
```

Comment:
We should have the publish script edit this file if it needs to be constantly updated.